crates/core/tedge_mapper/src/mapper.rs

```diff
@@ -31,12 +34,28 @@ pub(crate) fn mqtt_config(
 pub struct Mapper {
     client: mqtt_client::Client,
     converter: Box<dyn Converter<Error = ConversionError>>,
+    operations: Operations,
     _flock: Flockfile,
 }
 
 impl Mapper {
-    pub(crate) async fn run(&mut self) -> Result<(), MqttClientError> {
+    pub fn new(
+        client: mqtt_client::Client,
+        converter: Box<dyn Converter<Error = ConversionError>>,
+        operations: Operations,
+        _flock: Flockfile,
+    ) -> Self {
+        Self {
+            client,
+            converter,
+            operations,
+            _flock,
+        }
+    }
+
+    pub(crate) async fn run(&mut self, cloud: &str) -> Result<(), MqttClientError> {
         info!("Running");
+        let () = self.post_operations(cloud).await?;
```

Comment:
This doesn't look like a good idea...

Reply:
One point is sure is that this run for any cloud while it is C8Y specific.

I would rather extend the converter.